test/functional/zmq_test.py

```diff
@@ -107,8 +142,17 @@ def _zmq_test(self):
         hashZMQ = bytes_to_hex_str(body)
         msgSequence = struct.unpack('<I', msg[-1])[-1]
         assert_equal(msgSequence, blockcount + 1)
-
+        
+        msg = self.zmqSubSocket.recv_multipart()
+        topic = msg[0]
+        body = msg[1]
+        hashedZMQ = ""
+        assert_equal(topic, b"rawtx")
```

Comment:
I'd move this assert to immediately after the `topic = ...` line (like in the previous tests in this script).